dist/s_fixme

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
+
+# Find FIXME comments related to the ticket potentially indicated by the branch name.
+# The script only fails if the branch name follows a specific syntax and FIXME comments are found,
+# otherwise, the script exits without failing.
+
+# Retrieve the current branch name.
+branch_name=$(git rev-parse --abbrev-ref HEAD)
+
+# We expect the following syntax (case insensitive): wt-<digits>.
+regex="^(wt|WT|wT|Wt)-[0-9]+(-[a-zA-Z0-9-])?"
+if [[ ! $branch_name =~ $regex ]]; then
+    exit 0
+fi
+
+# Get what could be the ticket id.
+ticket_id=$(echo "$branch_name" | cut -d "-" -f-2)
+
+# Check if the suspected id only contain numbers.
+regex="^(wt|WT|wT|Wt)-[0-9]+$"
+if [[ ! $ticket_id =~ $regex ]]; then
```

Comment:
Is there any case where this can fail? To reach here we need to match against the regex on line 11 which guarantees the branch name is either `wt-1234` or `wt-1234-description`

Reply:
That is correct, the first check already guarantees what is being checked here. Removed in 67183b7ac30e9b33e82a73ea8c2e728376e25cb0.